pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3012,16 +3019,17 @@ do_file_io_operation(struct pt_regs *ctx, u32 event_id, u32 tail_call_id, bool i
         // missed entry or not traced
         return 0;
     }
+    del_args(event_id);
```

Comment:
Why the arg deletion should occur even if not traceable etc?

Reply:
I assume because the args now live in saved_args. So no need to keep them around in the "inbetween" map.